src/EditorFeatures/Test2/Workspaces/SymbolDescriptionServiceTests.vb

```diff
@@ -108,44 +96,6 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
             Await TestCSharpAsync(workspace, $"({FeaturesResources.local_constant}) int x = 2")
         End Function
 
-        <Fact>
```

Comment:
were these tests moved somewhere?

Reply:
Full comment is attached to 173f05c941f202237499d2a97db9e019e34984dc, but the summary: these are symbol description service tests. It's testing Quick Info behavior too not by running Quick Info code, but having the test harness copy/paste Quick Info code....and then just test the test harness. It's already tested too by Quick Info code, and that tests the *actual* product code.